api-extractor/src/DocItemLoader.ts

```diff
@@ -206,14 +206,14 @@ export default class DocItemLoader {
       return;
     }
 
-    return this.loadPackageIntoCache(packageJsonFilePath);
+    return this.loadPackageIntoCache(packageJsonFilePath, false);
```

Comment:
I added a second parameter as a flag to signal in the next step if this should be saved with an @types scope name. 
This flag is always false if loadPackageIntoCache() is called from the DocItemLoader().